src/main/kotlin/uk/gov/dluhc/printapi/service/aed/AnonymousElectorDocumentSearchService.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.dluhc.printapi.service.aed
+
+import org.springframework.data.domain.Page
+import org.springframework.data.domain.PageRequest
+import org.springframework.data.domain.Sort
+import org.springframework.data.domain.Sort.Direction.ASC
+import org.springframework.data.domain.Sort.Direction.DESC
+import org.springframework.stereotype.Service
+import uk.gov.dluhc.printapi.database.entity.SourceType.ANONYMOUS_ELECTOR_DOCUMENT
+import uk.gov.dluhc.printapi.database.repository.AnonymousElectorDocumentSummaryRepository
+import uk.gov.dluhc.printapi.dto.aed.AnonymousSearchSummaryDto
+import uk.gov.dluhc.printapi.mapper.aed.AnonymousSearchSummaryMapper
+import uk.gov.dluhc.printapi.service.EroService
+
+@Service
+class AnonymousElectorDocumentSearchService(
+    private val eroService: EroService,
+    private val anonymousElectorDocumentSummaryRepository: AnonymousElectorDocumentSummaryRepository,
+    private val anonymousSearchSummaryMapper: AnonymousSearchSummaryMapper,
+) {
+
+    fun searchAnonymousElectorDocumentSummaries(eroId: String): Page<AnonymousSearchSummaryDto> {
```

Comment:
Re: my comment above, this return type presumably means the controller needs to know what to do with a `org.springframework.data.domain.Page` ?? That doesnt feel right?

Reply:
Really? bloomin' eck! I wish we hadn't have done that in VCA 👎 
Never mind, that ship has sailed ... 😢

Reply:
Is that right @vishalgupta-valtech? The controller in VCA doesn't know about the `org.springframework.data.domain` package - we use integers to represent pages and pageSize etc.

The service class needs to import `Page` and `PageRequest`, but not the controller.

Reply:
Actually you're right but it doesn't explicitly need to import Page, hence my initial confusion. mmm...that's not ideal.

Reply:
VCA Controller 
<img width="570" alt="image" src="https://user-images.githubusercontent.com/73763010/234532073-1bb1d0ce-0cfa-452b-bad8-4c16f761f51a.png">

Reply:
Yes, I guess this is perhaps one of the bad things about kotlin (😬 , did I really say that? 🤣 )
Where kotlin can infer the type we dont need to have an explicit import, so this level of coupling gets somewhat lost

🤔

Reply:
Indeed, I was a bit quick to only examine the imports! 😆

Reply:
@NathanRussellValtech @matt-wills-valtech As suggested, pushed a code that introduces `AnonymousSearchSummaryPageDto` that now will handle all values from pagedResponse within service